refinery/ui/source/js/dashboard/controllers/analysis-deletion.js

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
I thought we agreed to this style guide for angularjs. https://mgechev.github.io/angularjs-style-guide/#controllers